weldx/schemas/weldx.bam.de/weldx/core/transformations/coordinate_system_hierarchy-1.0.0.yaml

```diff
@@ -9,115 +9,191 @@ description: |
       !<asdf://weldx.bam.de/weldx/tags/core/transformations/coordinate_system_hierarchy-1.0.0>
-        name: Coordinate system manager 0
-        root_system_name: base
-        subsystems: []
-        coordinate_systems:
-        - !<asdf://weldx.bam.de/weldx/tags/core/transformations/coordinate_transformation-1.0.0>
-          name: workpiece
-          reference_system: base
-          transformation: !<asdf://weldx.bam.de/weldx/tags/core/transformations/local_coordinate_system-1.0.0> {}
-        - !<asdf://weldx.bam.de/weldx/tags/core/transformations/coordinate_transformation-1.0.0>
-          name: tcp_wire
-          reference_system: workpiece
-          transformation: !<asdf://weldx.bam.de/weldx/tags/core/transformations/local_coordinate_system-1.0.0>
-            time: !<asdf://weldx.bam.de/weldx/tags/time/timedeltaindex-1.0.0>
-              values: !core/ndarray-1.0.0
-                data: [0, 29000000000]
-                datatype: int64
-                shape: [2]
-              start: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M0S}
-              end: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M29S}
-              min: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M0S}
-              max: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M29S}
-            orientations: !<asdf://weldx.bam.de/weldx/tags/core/variable-1.0.0>
-              name: orientations
-              dimensions: [c, v]
-              dtype: <f8
-              data: !core/ndarray-1.0.0
-                data:
-                - [1.0, 0.0, 0.0]
-                - [0.0, -1.0, -1.2246467991473532e-16]
-                - [0.0, 1.2246467991473532e-16, -1.0]
-                datatype: float64
-                shape: [3, 3]
-            coordinates: !<asdf://weldx.bam.de/weldx/tags/core/variable-1.0.0>
-              name: coordinates
-              dimensions: [time, c]
-              dtype: <f8
-              data: !core/ndarray-1.0.0
-                data:
-                - [5.0, 0.0, 2.0]
-                - [295.0, 0.0, 2.0]
-                datatype: float64
-                shape: [2, 3]
-        subsystem_names: []
-
-properties:
-  name:
-    type: string
+        name: CSM
+        graph: !<asdf://weldx.bam.de/weldx/tags/core/graph/di_graph-1.0.0>
+          root_node: !<asdf://weldx.bam.de/weldx/tags/core/graph/di_node-1.0.0>
+            name: R
+            attributes:
+              data: {}
+            edges:
+            - !<asdf://weldx.bam.de/weldx/tags/core/graph/di_edge-1.0.0>
+              direction: bwd
+              attributes:
+                defined: true
+                transformation: !<asdf://weldx.bam.de/weldx/tags/core/transformations/local_coordinate_system-1.0.0> {}
+              target_node: !<asdf://weldx.bam.de/weldx/tags/core/graph/di_node-1.0.0>
+                name: A
+                attributes:
+                  data: {}
```

Comment:
I guess the example is created using the debug code:
```python
from weldx import CoordinateSystemManager

csm = CoordinateSystemManager("R", "CSM")
csm.create_cs("A", "R")
csm.create_cs("B", "R")
csm.create_cs("C", "B")
```
looking at the csm plot output in a notebook the arrows point `R->A`, `R->B` etc.
In the schema should that not result in a `fwd` direction of the edges or am I mixing things up here?